.eslintrc.js

```diff
@@ -2,7 +2,10 @@ module.exports = {
   "extends": "eslint:recommended",
   "plugins": [],
   "parserOptions": {
-    "ecmaVersion": 2020
+    "ecmaVersion": 2019,
```

Comment:
Any particular reason for changing this to an earlier version?

Reply:
I was facing linter issues...weird that I'm not facing any issues now...I'll revert it back to 2020

Reply:
The version of eslint you were using was 3 major versions behind, which is why it didn't like the newest ecmaversion.

Reply:
yes...figured it out yesterday